applications/polkadot-mempool-explorer-v2.md

```diff
@@ -0,0 +1,499 @@
+
+  
+
+- <https://www.linkedin.com/in/dikhyantkrishnadalai/> Dikhyant Krishna
+
+  
+
+  
+
+## Development Roadmap :nut_and_bolt:
+
+  
+
+  
+
+### Overview
+
+  
+
+  
+
+-  **Total Estimated Duration:** 7-8 weeks
+  
+
+-  **Full-Time Equivalent (FTE):** 3
+
+  
+
+-  **Total Costs:** 9000 USD
+
+  
+
+  
+
+### Milestone 1
+
+  
+
+#### version 2 of mempool dashboard and polkadot js API tutorials
+
+
+  
+
+-  **Estimated Duration:** 7-8 weeks
+
+  
+
+-  **FTE:** 3
+
+  
+
+-  **Costs:** 9000 USD
+
+   
+|  Number | Deliverable                            | Specification                                                                                                                                                                                                                |
+| ------: | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **0a.** | License                                | APACHE 2                                                                                                                                                                                                                     |
+| **0b.** | Documentation                          | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how to view pending transactions in dashboard |
+| **0c.** | Testing and Testing Guide              | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                              |
+| **0d.** | Docker                                 | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.                                                                                                                |  |  |  |
+|      1. | new UI for desktop and mobile view and fix existing issues|                                                                                                                                            |
+|      2. | enhance the APIs and fix existing issues                                                                                                                                                                        |
+|      3. | enable CI/CD | spin up a server instance for hosting the backend and deploying the frontend at github pages                                                                                                                                                     |
+  | 4a | polkadot js API tutorials             | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how to run sample examples |
```

Comment:
Will this tutorial be a separate document somewhere, or more of a UI overlay/tooltips on the page itself?

Reply:
@semuelle it will be a series of tutorials something like this https://github.com/muddlebee/polkadot-js-beginner-series
hosted in a website rendering markdown as syntax like https://rust-unofficial.github.io/too-many-lists/index.html

Reply:
![image](https://user-images.githubusercontent.com/8139783/221169424-0880be09-3767-47b7-a71f-dbaae91b523b.png)

@semuelle updated this information in the proposal for better clarification